apps/api/src/common/env/env.schema.ts

```diff
@@ -18,6 +18,12 @@ Use the .optional() property if you are okay with a variable being omitted from
 
 */
 
+const validateSampleRate = (val: string | undefined) => {
```

Comment:
You can actually get rid of this, and use the `min`, `max` and `transform` functions of zod.